ui/src/app/services/sync.notifier.ts

```diff
@@ -1,26 +1,35 @@
 import { Injectable } from '@angular/core'
```

Comment:
Just to be clear, we are making a number of assumptions in this implementation:
- We get server updates only via the sync daemon
- We will only show a welcome message at most once in the lifetime of a UI
- We will only check for updates at most once in the lifetime of a UI
- If there is an opportunity to upgrade the OS, there is no need to indicate new apps available.

Each may be true in practice, now, but none are necessarily true. If any of the above assumptions later become false, it will be difficult to remember to change things here. It will also non self-documenting. It will be difficult for a new dev to figure out on their own why this is here/how this works without learning, from somewhere else, why the above assumptions are valid.